pkg/controller/replication/replication_controller.go

```diff
@@ -202,8 +202,7 @@ func (rm *ReplicationManager) updateRC(old, cur interface{}) {
 	oldRC := old.(*v1.ReplicationController)
 	curRC := cur.(*v1.ReplicationController)
 
-	// TODO: Remove when #31981 is resolved!
-	glog.Infof("Observed updated replication controller %v. Desired pod count change: %d->%d", curRC.Name, *(oldRC.Spec.Replicas), *(curRC.Spec.Replicas))
+	glog.V(4).Infof("Observed updated replication controller %v. Desired pod count change: %d->%d", curRC.Name, *(oldRC.Spec.Replicas), *(curRC.Spec.Replicas))
```

Comment:
Gate this as per @eparis suggestion and also remove the if clause below since we already log status updates in the [controller](https://github.com/kubernetes/kubernetes/blob/5ba21e83b9f3c50dab929ed6eb2ad5147fb0ff75/pkg/controller/replication/replication_controller_utils.go#L54)